frontend/src/scenes/session-recordings/player/PlayerConsole.tsx

```diff
@@ -41,52 +41,50 @@ export function PlayerConsole(): JSX.Element | null {
     return (
         <div className="console-log-container">
             <div className="console-log">
-                {areAllSnapshotsLoaded ? (
-                    orderedConsoleLogs.length > 0 ? (
-                        <AutoSizer>
-                            {({ height, width }: { height: number; width: number }) => (
-                                <div style={{ height: height, width: width, overflowY: 'scroll', paddingBottom: 5 }}>
-                                    {/* Only display the first 150 logs because the list ins't virtualized */}
-                                    {orderedConsoleLogs.slice(0, 150).map((log, index) => renderLogLine(log, index))}
-                                    <div>
-                                        {orderedConsoleLogs.length > 150 && (
-                                            <div className="more-logs-available">
-                                                While console logs are in beta, only 150 logs are displayed.
-                                            </div>
-                                        )}
-                                    </div>
-                                </div>
-                            )}
-                        </AutoSizer>
-                    ) : (
-                        <div
-                            style={{
-                                display: 'flex',
-                                flexDirection: 'column',
-                                height: '100%',
-                                justifyContent: 'center',
-                                alignItems: 'center',
-                                margin: 20,
-                            }}
-                        >
-                            <h3 style={{ textAlign: 'center' }}>There are no console logs for this recording</h3>
-
-                            <p className="text-muted" style={{ textAlign: 'center' }}>
-                                For logs to appear, the feature must first be enabled in <code>posthog-js</code>.
-                            </p>
-                            <LemonButton
-                                type="secondary"
-                                style={{ margin: '0 8px' }}
-                                href="https://posthog.com/docs/user-guides/recordings?utm_campaign=session-recording&utm_medium=in-product"
-                            >
-                                Learn more
-                            </LemonButton>
-                        </div>
-                    )
-                ) : (
+                {sessionPlayerDataLoading ? (
```

Comment:
The value of `sessionPlayerDataLoading` reflects if there's an active requests from playerData. Because player data is paged in, that means it flips from true to false on each request. If you watch the spinner while data is paging in, you'll see it flicker. Probably best to use something like "Has all data loaded".